cmd/start-amazon-cloudwatch-agent/start-amazon-cloudwatch-agent.go

```diff
@@ -92,8 +93,25 @@ func main() {
 		log.Fatalf("E! Cannot translate JSON config into TOML, ERROR is %v \n", err)
 	}
 	log.Printf("I! Config has been translated into TOML %s \n", tomlConfigPath)
+	echoTOML(tomlConfigPath)
 
 	if err := startAgent(writer); err != nil {
+		log.Printf("E! Error when starting Agent, Error is %v \n", err)
 		os.Exit(1)
 	}
 }
+
+func echoTOML(path string) {
```

Comment:
Adam already commented on the naming, which I agree with. I am mildly concerned about the `log.Fatal()` used here. I see above that there is a call directly to `os.Exit()`. Would that be preferable to causing a panic here?